### PR TITLE
add on-hold to journal

### PR DESCRIPTION
**Pull request by domgan**
* added on-hold quests (they were not shown at all)
* added small counter of quests
* fixed loading overlay (broken by me recently 😬)

image

### CODE REVIEW

File: src/views/journal/JournalComponent.tsx [hunk 1]
```diff
@@ -18,21 +26,39 @@ const JournalComponent = () => {
...
+                    >
+                        {label}
+                        {questCounts![state] > 0 && (
+                            <span className={style.count}>{questCounts![state]}</span>
+                        )}
+                    </button>
+                ))}
             </div>
-            {filteredQuests.map((quest) => (
+            {filteredQuests!.map((quest) => (
```
Comment (matfij) [inline src/views/journal/JournalComponent.tsx:61]:
```suggestion
            {filteredQuests?.map((quest) => (
```

File: src/views/journal/JournalComponent.module.scss [hunk 2]
```diff
@@ -25,11 +25,24 @@
...
         &:hover {
             background-color: $color-light;
             color: $color-dark;
         }
+
+        .count {
+            position: absolute;
+            top: 0;
+            right: 0;
+            color: white;
```
Comment (matfij) [inline src/views/journal/JournalComponent.module.scss:39]:
```suggestion
            color: $color-dark;
```

Comment (matfij) [reply src/views/journal/JournalComponent.module.scss:39]: or add some text-shadow for contrast + use standard `$color-light` color from out palette;

Comment (domgan) [reply src/views/journal/JournalComponent.module.scss:39]: I think it looks nice the way it is

Comment (domgan) [reply src/views/journal/JournalComponent.module.scss:39]: I tried text shadow before and it didn't look well but I'll try again

Comment (domgan) [reply src/views/journal/JournalComponent.module.scss]: Added something simple

**Review by matfij — APPROVED** [review]
LGTM

**Pull request merged**